controllers/deactivation/deactivation_controller.go

```diff
@@ -221,6 +282,21 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 	return reconcile.Result{}, nil
 }
 
+func setDeactivationTimestamp(ctx context.Context, r *Reconciler, mur *toolchainv1alpha1.MasterUserRecord,
+	userSignup *toolchainv1alpha1.UserSignup, deactivationTimeout time.Duration, logger logr.Logger) error {
+	scheduledDeactivationTime := (*mur.Status.ProvisionedTime).Add(deactivationTimeout)
+	if userSignup.Status.ScheduledDeactivationTimestamp == nil || !userSignup.Status.ScheduledDeactivationTimestamp.Time.Equal(scheduledDeactivationTime) {
+		ts := v1.NewTime(scheduledDeactivationTime)
+		userSignup.Status.ScheduledDeactivationTimestamp = &ts
+		if err := r.Client.Status().Update(ctx, userSignup); err != nil {
```

Comment:
Use the status updater here too.

Reply:
I've removed the function all together, after replacing the code to use statusUpdater it turned out to just be 2 lines of code which isn't worth putting in its own function.